drivers/gles2/shaders/scene.glsl

```diff
@@ -2166,7 +2169,20 @@ FRAGMENT_SHADER_CODE
 #ifdef USE_VERTEX_LIGHTING
 	//vertex lighting
 
-	specular_light += specular_interp * specular_blob_intensity * light_att;
+#ifndef BASE_PASS
+	// Approximate Fresnel term for vertex lights
+	// This is needed because the vertex light pass doesn't have access to metallic, specular, or albedo and the fragment pass does not have access to light direction
+	float cNdotV = clamp(dot(normal, eye_position), 0.0, 1.0);
+	vec3 f0 = F0(metallic, specular, albedo);
+	vec3 F = mix(vec3(SchlickFresnel(cNdotV)), vec3(1.0), f0);
+	const vec4 c0 = vec4(-1.0, -0.0275, -0.572, 0.022);
+	const vec4 c1 = vec4(1.0, 0.0425, 1.04, -0.04);
+	vec4 r = roughness * c0 + c1;
+	float a004 = min(r.x * r.x, exp2(-9.28 * cNdotV)) * r.x + r.y;
+	vec2 env = vec2(-1.04, 1.04) * a004 + r.zw;
+#endif
+
+	specular_light += specular_interp * (env.x * F + env.y) * specular_blob_intensity * light_att;
```

Comment:
I'm not super happy with this change. I originally added this in https://github.com/godotengine/godot/pull/29909 and then I removed it later when I "fixed" the blinn function in https://github.com/godotengine/godot/pull/33836.

Now it is clear that we need a Fresnel with Blinn, I have reintroduced it. This copies the split sum approach used by IBL, so it isn't a perfectly accurate Fresnel term for specular lobes (it doesn't account for light direction). But it is better than nothing. We can't get an accurate Fresnel term in the vertex shader because we don't have metallic, specular, or albedo until the fragment stage. We could set metallic and specular during the vertex stage, but we still need albedo.